zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -606,6 +609,120 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
     websocketMsgSrv.getInterpreterBindings($scope.note.id);
   };
 
+  $scope.toggleSelection = function(paragraphId) {
+    let paragraphs = $scope.selectedParagraphsIds;
+    if (paragraphs.has(paragraphId)) {
+      paragraphs.delete(paragraphId);
+    } else {
+      paragraphs.add(paragraphId);
+    }
+  };
+
+  $scope.clearSelection = function() {
+    if ($scope.selectedParagraphsIds !== null) {
+      $scope.selectedParagraphsIds.clear();
+    }
+  };
+
+  $scope.isSelectionMode = function() {
+    if ($scope.selectedParagraphsIds === null || $scope.selectedParagraphsIds.size === 0) {
+      return false;
+    }
+    return true;
+  };
+
+  $scope.showConfirmDialog = function(dialogMessage, action) {
+    BootstrapDialog.confirm({
+      closable: true,
+      title: '',
+      message: dialogMessage,
+      callback: function(result) {
+        if (result) {
+          action();
+        }
+      },
+    });
+  };
+
+  $scope.selectedParagraphsAction = function(type, requestConfirm) {
+    let noteId = $scope.note.id;
+    let action;
+    let broadcastAction = function(data) {
+      return () => $scope.$broadcast('multiplyAction', type, $scope.selectedParagraphsIds, data);
```

Comment:
replace "multiplyAction" with "multipleAction" in all places

Reply:
Thanks Tagar. I will fix it.